include/seastar/util/shared_token_bucket.hh

```diff
@@ -160,6 +160,10 @@ public:
         _rovers.release(tokens);
     }
 
+    void refund(T tokens) noexcept {
```

Comment:
Reposting a comment by @xemul which he wrote in https://github.com/michoecho/seastar/commit/b0ec97d4117481101ec464f5d8e18a2bb7124827#r150664671. (I created this PR just to anchor his comment to something. Comments attached to commits are hard to find)"

> We had some experience with returning tokens to bucket. This didn't work well, mixing time-based replenish with token-based replenish had a weird effect. Mind reading https://github.com/scylladb/seastar/pull/1766, specifically https://github.com/scylladb/seastar/pull/1766#issuecomment-1715050516 and https://github.com/scylladb/seastar/pull/1766#issuecomment-1672923672 comments for details. If we're going to go with this fix, we need some justification of why we don't step on the same problem again